src/applications/pact-act/containers/questions/TernaryRadios.jsx

```diff
@@ -1,11 +1,8 @@
 import React, { useState } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import {
-  VaButtonPair,
-  VaRadio,
-  VaRadioOption,
-} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import _ from 'lodash';
```

Comment:
We should be able to use something like

```
import { snakeCase } from 'lodash'
```

No need to import the whole library if the file doesn't need it.